__test__/cypress/integration/basic-campaign-e2e.test.js

```diff
@@ -18,24 +23,36 @@ describe("End-to-end campaign flow", () => {
     // Fill out basics
     cy.get("input[data-test=title]").type(campaignTitle);
     cy.get("input[data-test=description]").type(campaignDescription);
-    cy.get("input[data-test=dueBy]").click();
+
+    // DatePicker is difficult to interact with as its components have no ids or classes
+    // So set date directly for test and ignore picker  (interaction commented-out below)
+
+    // cy.get("input[data-test=dueBy]").focus();
+    cy.get("input[data-test=dueBy]").type(campaignDueBy, { force: true });
+    cy.get("button")
+      .contains("Cancel")
+      .click();
+    cy.wait(200);
+    cy.pause();
+    cy.get("input[data-test=dueBy]").then(input => {
+      input.val(campaignDueBy);
+    });
```

Comment:
@schuyler1d If you have a chance, check this out in the Cypress runner. Something is setting the dueBy back to 1/1/2000 after I set it programmatically. This code sets it once by typing into the input before/while the picker displays and then is reset when the picker renders. After the picker is closed, it sets it again by directly accessing the form input and then in reset when the form submit is clicked.

I can't see what may be resetting the value to 1/1/2000. Any ideas?

Reply:
Resolved.